### PR TITLE
Fix update

### PR DESCRIPTION
**Pull request by Tropicalista**

### CODE REVIEW

**Comment by Tropicalista** [comment]
This should be a patch for updating 3.0.0 beta, not 2.1.0

File: workbench/patches/3.0.0-beta/Update.cfc [hunk 1]
```diff
@@ -197,11 +196,15 @@ component implements="contentbox.model.updates.IUpdate"{
 
 	private function updatePermissions(){
 		// Update Old Permissions
-		var perm = permissionService.findWhere( { name="LAYOUT_ADMIN" } );
+		var perm = permissionService.findWhere( { name="THEME_ADMIN" } );
 		if( !isNull( perm ) ){
 			perm.setName( "THEME_ADMIN" );
 			perm.setDescription( "Ability to manage themes, default is view only" );
 			permissionService.save( entity=perm, transactional=false );
+		}else{
```
Comment (lmajano) [inline workbench/patches/3.0.0-beta/Update.cfc]: I don't get this part @Tropicalista  It seems to re-insert if the theme admin perm is not there, else try to look for the other on.  Everything else looks great except this portion that doesn't make sense to me.

**Comment by Tropicalista** [comment]
Luis, this update is for people running contentbox 2.1 or for people running 3.0 beta?

The line you commented is responsible to manage permission about THEME_ADMIN that is not present in 2.1. I thought this update is for who's running 2.1, so I add theme_admin permission.

Could you clarify this and I will fix the error? 

Thanks.

File: workbench/patches/3.0.0-beta/Update.cfc [hunk 2]
```diff
@@ -197,8 +196,12 @@ component implements="contentbox.model.updates.IUpdate"{
...
 	private function updatePermissions(){
 		// Update Old Permissions
-		var perm = permissionService.findWhere( { name="LAYOUT_ADMIN" } );
+		var perm = permissionService.findWhere( { name="THEME_ADMIN" } );
 		if( !isNull( perm ) ){
+			perm.setDescription( "Ability to manage themes, default is view only" );
+			permissionService.save( entity=perm, transactional=false );
+		}else{
+			var perm = permissionService.findWhere( { name="LAYOUT_ADMIN" } );
 			perm.setName( "THEME_ADMIN" );
```
Comment (Tropicalista) [inline workbench/patches/3.0.0-beta/Update.cfc]: Rename the permission

**Comment by lmajano** [comment]
I finally got it,  Merged manually. Thank you, please try the updater again.

**Pull request closed without merging**